containers/daap-python-base/src/var/task/curated_data/curated_data_loader.py

```diff
@@ -80,3 +81,182 @@ def ingest_raw_data(self, raw_data_table: QueryTable, extraction_timestamp: str)
             database_name=self.curated_data_table.database,
             table_name=self.curated_data_table.name,
         )
+
+    def create_new_major_version_data_product_table(self, is_updated_table):
```

Comment:
I think this could be split into two functions, since there's nothing in common between the two branches

i.e. we could have something like

```python
def copy_table_from_previous_major_version(self)
#...

def create_table_and_unload_from_previous_major_version(self)
#...
```

Reply:
Thinking about this i'd rather keep as one. Keeps the call more simple as otherwise we'd need to conditionally run either method from the `CuratedDataLoader` class